lib/compact_tweet_view.dart

```diff
@@ -40,6 +42,7 @@ class CompactTweetView extends StatelessWidget {
   /// If set to true a chewie/video_player will be used in a Tweet containing a video.
   /// If set to false a image placeholder will he shown and a video will be played in a new page.
   final bool useVideoPlayer;
+  final Function onTapImage;
```

Comment:
With the current code, I think it should be 
`final onTapImage onTapImage;` instead of `final Function onTapImage;`

since you have:
 `typedef onTapImage = void Function(List<String> allPhotos, int photoIndex, String hashcode);`

Like in this example:
```dart
typedef Compare = int Function(Object a, Object b);

class SortedCollection {
  Compare compare;

  SortedCollection(this.compare);
}
```

Same in all other Tweet views with the typedef you added ;)

Reply:
I'm not entirely sure what you want me to do? :(

I'm not super familiar with using typedef